src/reaper_osara.cpp

```diff
@@ -3756,6 +3778,53 @@ void cmdMoveToPrevItem(Command* command) {
 	}
 }
 
+void reportNudgeAndZoomStep(double nudgeTime) {
+	TimeFormat tf = TF_MILSEC;
+	if (nudgeTime >= 1.0) tf = TF_ROUNDSEC;
+	outputMessage(formatTime(nudgeTime, tf));
+}
+
+void nudgeAndZoomStep(int direction) {
+	// direction = +1 → zoom out (increase nudge time), -1 → zoom in (decrease nudge time)
+	double currentPPS = GetHZoomLevel();
+	if (currentPPS <= 0.0) {
+		outputMessage(translate("invalid zoom level"));
```

Comment:
Should this ever happen? If not, we could just assert here, or if you want to be extra paranoid, return without reporting anything. Otherwise, translators have to translate something that will never be used.